apps/sms/views/conversation/js/conversation.js

```diff
@@ -751,6 +785,13 @@ var ConversationView = {
     this.cleanFields();
     this.initRecipients();
     this.updateComposerHeader();
+
+    Compose.setupLock({
+      canSend: () => this.hasValidRecipients(),
+      // Null means that we want to use default type detection strategy.
+      getType: () => this.hasEmailRecipients() ? 'mms' : null
```

Comment:
Yeah it's more readable, but I didn't want to do this because in case ```this.hasEmailRecipients()``` returns ```false```, ```forceType``` returns boolean. Not a big deal for JS, but still :)